tests/test_brain_large_speedup.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""Test improved calculation of log_zinb_forward for better performance on brain_large."""
+
+from scipy.sparse import random
+import torch
+import torch.nn.functional as F
+
+batch_size = 128
+num_genes = 720
+
+x = torch.from_numpy(random(batch_size, num_genes,
```

Comment:
This is a hacky way of simulating data but is probably fine. If you're curious, we have a simulation API I believe here, which would be probably more appropriate
https://github.com/YosefLab/scVI/blob/master/scvi/dataset/synthetic.py

Reply:
Thanks for the pointer, I've updated to use `SyntheticDataset` now